### DISCUSSION TITLE
欢迎来到蓝鲸权限中心讨论组 !

### DISCUSSION THREAD

**Discussion by wklken** [TencentBlueKing/bk-iam-saas#43] [Announcements]
## 👋  欢迎

这是一个蓝鲸权限中心的讨论组

讨论组用于:
- 讨论一些你关心的问题, 关于产品/模型/实现等
- 抛出你的疑问
- 分享你的想法
- 给出你的建议

讨论组不用于:
- bug report, 请通过issues提bug
- 部署/生产环境等问题解决, 请通过蓝鲸社区讨论组处理

**Comment by wqczj** [comment]
蓝鲸的权限管理中，对模型实例的查看的颗粒度能否到实例级别。目前社区版的模式实例查看好像不是实例级别的,
